source/channels/format-messages.rst

```diff
@@ -18,13 +18,131 @@ Format messages
+
+.. |hide-formatting-icon| image:: ../images/format-letter-case_F0B34.svg
+  :height: 24px
+  :width: 24px
+  :target: https://mattermost.com/deploy
+  :alt: Hide formatting options in the message formatting toolbar using the Show/Hide Formatting icon.
+
+.. |preview-icon| image:: ../images/eye-outline_F06D0.svg
+  :height: 24px
+  :width: 24px
+  :target: https://mattermost.com/deploy
+  :alt: Review your message text formatting using the Show/Hide preview icon in the message formatting toolbar.
+
+Use the messaging formatting toolbar
+------------------------------------
+
+From Mattermost v7.0, you can format your messages in Mattermost using the message formatting toolbar without having to specify any Markdown syntax.
+
+.. image:: ../images/message-formatting-toolbar.png
+  :alt: The message formatting toolbar, available from Mattermost v7.0, doesn't require Markdown syntax, and makes formatting message text fast and easy.
+
+The message formatting toolbar offers the following formatting options:
+
++------------------------------------------------------------+--------------------+
+| **Formatting option**                                      | **Icon**           | 
++============================================================+====================+
+| Bold, italicize, or strike out text                        | |bold-icon|        |
+|                                                            | |italics-icon|     | 
+|                                                            | |strikeout-icon|   | 
++------------------------------------------------------------+--------------------+
+| Add headings, links, or attachments                        | |headings-icon|    |
+|                                                            | |links-icon|       | 
+|                                                            | |attachments-icon| |
++------------------------------------------------------------+--------------------+
+| Format a numbered list, a bulleted list, quoted text, or   | |numbered-icon|    | 
+| text as code                                               | |bullets-icon|     | 
+|                                                            | |quotes-icon|      |
+|                                                            | |code-icon|        | 
++------------------------------------------------------------+--------------------+
+| Add emojis or GIFs                                         | |emoji-icon|       | 
++------------------------------------------------------------+--------------------+
+
+Review how your message formatting will look when the message is sent by selecting the **Show/Hide Preview** |preview-icon| icon. Return to your draft message by selecting the icon again.
+
+You can hide the formatting options by selecting the **Show/Hide Formatting** |hide-formatting-icon| icon. Select the icon again to display the formatting options.
+
+.. tip::
```

Comment:
If users had Enable Post Formatting set to off prior to an upgrade to v7.0, they may never see the Markdown formatting as described in this tip. Consider removing to avoid confusion.

Reply:
@cwarnermm They will see formatting options irrespective of their `` Enable Post Formatting`` setting, it doesn’t affect the Advanced text editor at all.
@matthewbirtch Do we need to do something here

Reply:
According to the [product documentation](https://docs.mattermost.com/channels/channels-settings.html#enable-post-formatting), the Enable Post Formatting option controls whether post formatting is rendered. When On, posts will be rendered with Markdown formatting. When Off, the raw text will be shown.

Currently, on the Community Server, using the message formatting toolbar options, I see the Markdown syntax because I know I had this setting enabled previously. But if a user had this setting disabled, I'm wondering whether they'd see the syntax. If they don't see it, I don't necessarily want to provide this level of detail in the docs around syntax.

Reply:
@esethna has confirmed that the Enable Post Formatting option is being deprecated in favour of the new message formatting toolbar. I'll update this PR based on this information.